src/math-nodeview.ts

```diff
@@ -38,7 +38,7 @@ export class MathView implements NodeView, ICursorPosObserver {
 	private _getPos: (() => number);
 
 	// nodeview dom
-	dom: HTMLElement;
+	dom?: HTMLElement;
```

Comment:
My TSC complained at this from the delete. There might be a better way to do this?